addons/mail/static/src/models/rtc_peer_connection.js

```diff
@@ -20,6 +20,28 @@ registerModel({
             const transceivers = this.peerConnection.getTransceivers();
             return transceivers[this.messaging.rtc.orderedTransceiverNames.indexOf(trackKind)];
         },
+        /**
+         * Sets the direction of the video transceiver of a given session.
+         *
+         * @param {boolean} allowReception
+         */
+        setVideoReceiverActivity(allowReception) {
+            const rtc = this.rtcSession.rtcAsConnectedSession;
+            if (!rtc) {
+                return;
+            }
+            const transceiver = this.getTransceiver('video');
+            if (!transceiver) {
+                return;
+            }
```

Comment:
Looks like this should be guarded before calling this method, because this method is not applicable in these conditions.